src/routes/v1/TrendingCompaniesRoute.ts

```diff
@@ -0,0 +1,16 @@
+
+import express, { Express } from 'express'
+import TrendingCompaniesController from '../../controllers/TrendingCompanies.controller'
+
+
+const trendingCompaniesRoute = (app: Express) => {
+    const TrendingController = new TrendingCompaniesController()
+    const router = express.Router()
+
+    //Someone please correct this cause I don't know what to do from here
+    router.get('/:TrendingCompanies',async (req,res) => {
+        await TrendingController.getTrendingCompanies(req,res);
+    })
+}
+
```

Comment:
need to add this line to define the endpoint. At the moment, the endpoint is `localhost:port/:TrendingCompanies` -> require a parameter. We want it to be `localhost:port/api/v1/trendingCompanies`
```suggestion
app.use("/api/v1/trendingCompanies", router)
```